webapp/reviews/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+
+from reviews.models import *
```

Comment:
Because, there's only one model, we don't really need to import the entire space